app/containers/SongDetailContainer/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { useParams } from 'react-router-dom';
+import { get, isEmpty } from 'lodash';
+import styled from 'styled-components';
+import T from '@components/T';
+import saga from './saga';
+import { selectItuneData } from '../SongContainer/selectors';
+import { selectTrackId, selectSingleItune } from './selectors';
+import { requestGetSingleItune } from './reducer';
+import { AnyAction } from '@reduxjs/toolkit';
+
+const DetailsCard = styled.div`
+  border: 1px solid black;
+  border-radius: 0.65rem;
+  padding: 1rem;
+  width: 70%;
+  height: 15rem;
+  margin: auto;
+  margin-top: 3rem;
+
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  justify-content: center;
+`;
+
+const SongImage = styled.img`
+  width: 7rem;
+  height: 7rem;
+  border-radius: 50%;
+  object-fit: cover;
+`;
+
+const SongDetails = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`;
+
+const PlayButton = styled.button`
+  padding: 0.75rem 1.5rem;
+  border: none;
+  border-radius: 0.5rem;
+  font-size: 1rem;
+  font-weight: 600;
+  background-color: #f94144;
```

Comment:
we don't define colours statically. Instead define it in the themes colours file and import them

```css

import colors from "./themes";
......
background-colour: ${colors.white};
```